src/SearchInput/SearchInput.tsx

```diff
@@ -51,6 +66,8 @@ export const SearchInput: FC<SearchInputProps> = ({
   searchList,
   onFound,
   resultsRelativePosition = false,
+  showCaret,
+  hasBorder,
```

Comment:
you should default them to false so they don't break any existing functionality

Reply:
so i've made the props optional in TS and then if they're not passed through they'll be evaluated as falsey anyway, so not sure they need the default prop putting on, but happy to if it's must :)